src/components/__tests__/OnboardingFlow.test.js

```diff
@@ -2,46 +2,71 @@ import React from 'react'
 import { shallow } from 'enzyme'
 import Typography from '@material-ui/core/Typography'
 import Button from '@material-ui/core/Button'
-import onboarding1 from 'src/assets/onboarding/cattabs.svg'
-import onboarding2 from 'src/assets/onboarding/squadcat.svg'
-import onboarding3 from 'src/assets/onboarding/adcat.svg'
+import catTabs from 'src/assets/onboarding/cattabs.svg'
+import squadCat from 'src/assets/onboarding/squadcat.svg'
+import adCat from 'src/assets/onboarding/adcat.svg'
+
+const getMockProps = () => ({
+  showMissionSlide: false,
+  onboarding: {
+    steps: [
+      {
+        imgName: 'cattabs',
+        title: 'Your tabs are doing great things',
+        subtitle: 'Now, every tab you open supports cats in need.',
```

Comment:
Use markdown in tests and assert on Typography variants.

Reply:
That works. In that case, maybe mock Markdown and assert it's receiving the prop value as children? Basically, looking to have a test of "this component knows how to use markdown".